### ISSUE TITLE
Espaces C^k

### ISSUE THREAD

**Issue by boisgera** [boisgera/CDIS#46]
Pfff ca va être laborieux ... Au-delà des notations:

  - C(A) ou C^0(A) ne pose pas de pb tant qu'on a une structure topologique

  - toute la théorie du calcul diff se fait dans des ouverts ; c'est donc C^k(U) pour U ouvert qui a "naturellement" du sens

  - bien sûr dans le cas d'une variable, il n'est pas difficile de définir dérivée à gauche ou à droite et donc C^k(I) pour I intervalle arbitraire. Ca tombe bien, parce que c'est équivalent à avoir un prolongement C^k sur un ouvert, donc on raccroche avec l'étape précédente.

  - dans le cas non-ouvert dans R^n, disons compact pour faire simple, si K est l'adhérence de son intérieur (ok si $K$ est compact à bord C^1 par exemple), on s'en sort avec le sous-ensemble des C^k sur l'intérieur de K dont les dérivées partielles sont uniformément continues (et / ou prolongeable sans pb sur le bord). Pas exactement trivial et de toute façon, certainement pas abordé avant le chapitre Calc Diff III, tout à la fin.

Sans avoir tout vérifié, je pense qu'à l'heure actuelle (d8425da6bf302782fd59e587e89cc79be151aa1d) le volet équa diff passe à travers
les gouttes (les cas non-ouvert se limitent à une dimension -- le temps -- où dans le cas contraire seule la continuité est utilisée) ... à regarder de plus près.

**Comment by paulinebernard** [comment]
Oui ok je vais regarder de plus près où ça intervient. De toute façon, en réf à ton autre commentaire sur la nécessité d'avoir l'intervalle de définition non réduit à t_0, je pense que je vais exiger pour parler de solution que l'intervalle I 
-soit d'intérieur non vide
- et contienne t_0 
(donc pas forcément des deux côtés de t_0), et demander la dérivabilité et l'équa diff seulement sur l'intérieur de l'intervalle.

**Comment by paulinebernard** [comment]
Je suis en train de faire les modif et je maintiens le coup d'"intérieur non vide"  ou "non réduit à un point" mais par contre je me dis qu'écrire x \in C^1( int I, \R^n) plutôt que x \in C^1( I, \R^n) me semble introduire plus de complexité que nécessaire. Après tout ils doivent connaître bien les fonctions réelles et dire x derivable sur I avec I fermé ne devrait pas leur poser des problèmes existentiels si ? Après tout c'est bien défini dans les 3 première phrases de wikipedia sur la dérivabilité.

Donc j'hésite à aller dans ce sens. Qu'en penses tu?

En plus sinon dans l'équation intégrale, j'écris l'intégrale de \dot x alors que \dot x n'existe pas forcément au bord en t_0

**Comment by boisgera** [comment]
Oui, le cas intervalle fermé ne devrait pas poser de pb majeur et on en parlera avant de toute façon ; par exemple, en calcul intégral, pour le théorème fondamental du calcul (si je suis la dérivée d'une fct sur [a, b], j'y suis intégrable). C'est juste le point 4 de la liste ma remarque initiale qui est pénible à mon avis, et je ne crois pas que tu y sois exposée.

**Comment by boisgera** [comment]
Après il y a une différence entre comprendre ce qu'est une fonction continûment dérivable sur [a, b] et savoir jouer dans l'espace C^1([a, b]). Le premier point, ca devrait être sans pb ; le second, pas clair, parce que je vais (volontairement) ne pas parler des structures aussi longtemps que possible.

**Comment by boisgera** [comment]
Ooops.

**Comment by paulinebernard** [comment]
oui je suis d'accord sur le fait que savoir jouer avec l'espace topologique est différent mais justement dans mon cours je ne le fais jamais, c'est juste une notation pour moi, pas de structure assiciée.
La seule partie "analyse fonctionnelle" que je fais c'est dans Cauchy Lipschitz mais seulement dans l'espace des fonctions continues sur un intervalle compact. Donc pas de souci pour les dérivées sur l'intervalle I, je remets comme avant.
